docs/topics/ref_shared-messaging-work-queue-resources.adoc

```diff
@@ -0,0 +1,10 @@
+[id='shared-messaging-work-queue-resources_{context}']
+= Shared Messaging Work Queue Resources
+
+To learn more about messaging patterns and Red{nbsp}Hat JBoss A-MQ for OpenShift, follow the links below:
+
+* link:http://www.enterpriseintegrationpatterns.com/patterns/messaging/Introduction.html[Enterprise Integration Patterns: An overview of messaging service concepts^]
+
+* link:http://docs.oasis-open.org/amqp/core/v1.0/amqp-core-messaging-v1.0.html[AMQP 1.0 Messaging Protocol specifications^]
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_jboss_a-mq/6.3/html-single/red_hat_jboss_a-mq_for_openshift/index[Documentation on Red{nbsp}Hat JBoss A-MQ 6.3 for OpenShift^]
```

Comment:
Did you want to link to docs for version 6 or 7?

Reply:
I believe the booster uses Versions 6.3, so I linked version 6 docs here.

Reply:
OK. (TBH, I think that's terrible, but that's out of scope here :-) )

Reply:
We will change to AMQ Broker 7 soon.  The image for it was released recently.

Reply:
Great!